corehq/ex-submodules/casexml/apps/stock/migrations/0012_auto__chg_field_stocktransaction_subtype.py

```diff
@@ -0,0 +1,68 @@
+# encoding: utf-8
+import datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        
+        # Changing field 'StockTransaction.subtype'
+        db.alter_column(u'stock_stocktransaction', 'subtype', self.gf('casexml.apps.stock.models.TruncatingCharField')(max_length=20, null=True))
```

Comment:
Can't remember how I handled this in the past with South. I think Alembic (SQLAlchemy) uses the super class in migrations (would be `CharField` in this case) unless you take special action. Seems like that would be ideal here.